microsoft-edge/webview2/how-to/winrt-from-js.md

```diff
@@ -0,0 +1,336 @@
+
+1. In the **Search** box, enter **WebView2**.
+
+1. Click the **Microsoft.Web.WebView2** card.  Detailed info appears in the middle area of the window.
+
+1. In the **Version** drop-down, select a **prerelease** version of the WebView2 SDK.  Note which version number you select.
+
+1. Click the **Install** button:
+
+   ![NuGet Package Manager, selecting the WebView2 SDK package, for the WinRTAdapter project.](winrt-from-js-images/pkg-mgr-wv2-sdk.png)
+
+The WebView2 prerelease SDK is now installed for the **WinRTAdapter** project.
+
+
+<!-- =============================================== -->
+## Step 5. Install WebView2 prerelease SDK, for webview2_sample_uwp project
+
+In the **webview2_sample_uwp** project, install the same prerelease version of the WebView2 SDK as you installed for the **WinRTAdapter** project, as follows:
+
+1. In Solution Explorer, right-click the **webview2_sample_uwp** project, and then select **Manage NuGet Packages**.  The NuGet Package Manager window opens.
+
+1. Select the **Include prerelease** checkbox.
+
+1. In the **Search** box, enter **WebView2**.
+
+1. Click the **Microsoft.Web.WebView2** card.  Detailed information appears in the middle area of the window.
+
+1. In the **Version** drop-down, select a **prerelease** version of the WebView2 SDK.  This needs to be the same version as for the **WinRTAdapter** project.
+
+1. Click the **Install** button.
+
+   The screen should look similar to the above section, except that now, **NuGet Package Manager** is open for the **webview2_sample_uwp** project instead of the **WinRTAdapter** project.
+
+The WebView2 prerelease SDK is now installed for the **webview2_sample_uwp** project.
+
+
+<!-- =============================================== -->
+## Step 6. Generate source code for selected host API items
+
+Next, configure the WebView2 WinRT JS Projection tool, to incorporate the WinRT classes that you want to use.  This generates source files that will then be compiled, rather than adding a `#` include of entire header files.     Generating code for these API items will enable your web-side JavaScript code to call these API items.
+
+In the example steps below, we'll specify two `Windows` namespaces, and the WebView2 WinRT JS Projection tool will generate source code for only those namespaces:
+*  `Windows.System.UserProfile` <!-- why add Windows.System.UserProfile? -->
+*  `Windows.Globalization.Language`
+
+Later, when the sample app is running, you'll call these API items from the **Console** of DevTools, to demonstrate that these specified host-side API items can be called from web-side code.
+
+In this walkthrough, specify two `Windows` namespaces, as follows:
+
+1. In Solution Explorer, right-click the **WinRTAdapter** project, and then select **Properties**.  The **WinRTAdapter Property Pages** dialog opens.
+
+1. On the left, select **Common Properties** > **WebView2**.
+
+1. Set **Use the WV2WinRT tool** to **Yes**.
```

Comment:
"Use WebView2 WinRT APIs" should be set to "No". This is so the SDK doesn't copy the WebView2 WinRT component to the project's output. This `WinRTAdapter` project isn't calling any WebView2 WinRT APIs, so it doesn't need the WinRT component.